### ISSUE TITLE
Add TypeScript support

### ISSUE THREAD

**Issue by opensas** [sveltejs/svelte-repl#130]
Now that official TypeScript support for Svelte has been announced, It would be very convenient to also add it to the Repl.

The Repl has become a very useful tool, not only to showcase applications and share code snippets, but also to report issues.

**Comment by pngwn** [comment]
This is not a simple change, with the way things are setup. But I have been looking into this and other improvements over the past week.

**Comment by opensas** [comment]
Great to know, it really is a very useful tool.

**Comment by bradphelan** [comment]
This would be a great addition

**Comment by samuelandert** [comment]
Indeed, this is one of the most interesting and anticipated REPL features.

**Comment by danawoodman** [comment]
Will this be considered by the core team? Would be pretty rad 😸

**Comment by vinayakkulkarni** [comment]
Is there any way we, the community, can help contribute in building the REPL using TS ?

**Comment by hperrin** [comment]
Is there a way to at least strip type declarations from a Svelte file, then we could use that?
